File: Calculator.c
#include <stdio.h>
void main()
{
        int a, b, c, d, e, f;
        int g;
        char choice;

        printf("enter the first number:");
        scanf("%d", &a);
        printf("choose the operation:\n");
        printf("1.addition\n2.substraction\n3.multiplication\n4.division\n");
        scanf("%s", &choice);
        printf("enter the second number:");
        scanf("%d", &b);

        c = a + b;
        d = a - b;
        e = a * b;

        switch (choice)
        {

        case '+':
                printf("the solution is: %d", c);
                break;
        case '-':
                printf("the solution is: %d", d);
                break;
        case '*':
                printf("the solution is: %d", e);
                break;
        case '/':
                if (b != 0)
                {
                    f = a / b;
                printf("The solution is: %d ", f);
                }
                else
                {
                        printf("the number cannot be divided");
                }
                break;

        default:
                printf("\nthe operator is not found");
        }
}

File: Calculator.java
import java.util.Scanner;

public class Calculator{
    public static void main(String[] args) {
        int a, b, c, d, e, f;
        int g;
        char choice;
        Scanner in = new Scanner(System.in);
        System.out.println("enter the first number:");
        a = in.nextInt();
        System.out.println("choose the operation:\n");
        System.out.println("1.addition\n2.substraction\n3.multiplication\n4.division\n");
        choice = in.next().charAt(0);
        System.out.println("enter the second number:");
        b = in.nextInt();

        c = a + b;
        d = a - b;
        e = a * b;

        switch (choice)
        {

        case '+':
                System.out.println("the solution is: "+ c);
                break;
        case '-':
                System.out.println("the solution is: "+ d);
                break;
        case '*':
                System.out.println("the solution is: "+ e);
                break;
        case '/':
                if (b != 0)
                {
                    f = a / b;
                System.out.println("The solution is: "+ f);
                }
                else
                {
                        System.out.println("the number cannot be divided");
                }
                break;

        default:
                System.out.println("\nthe operator is not found");
        }
    }
}